src/mindlessgen/qm/xtb.py

```diff
@@ -201,3 +228,20 @@ def get_xtb_path(binary_name: str | Path | None = None) -> Path:
             xtb_path = Path(which_xtb).resolve()
             return xtb_path
     raise ImportError("'xtb' binary could not be found.")
+
+
+def check_ligand_uhf(ati: np.ndarray, charge: int) -> bool:
+    """
+    Check if the remaning number of electrons after the f electrons are removed is even.
+    """
+    nel = 0
+    f_electrons = 0
+    for atom in ati:
+        nel += atom + 1
+        if atom in get_lanthanides():
+            f_electrons += atom - 56
+    # Check if the number of the remaning electrons is even
+    test_rhf = nel - f_electrons - charge
+    if not test_rhf % 2 == 0:
```

Comment:
Just `Raise ValueError` here, and do not return anything.

Reply:
But still return true in the end or just nothing?

Reply:
No, just no `return` statement at all. We just want to check for errors.